org.qgis.qgis.json

```diff
@@ -0,0 +1,459 @@
+            ],
+            "sources" : [
+                {
+                    "type" : "git",
+                    "url" : "git://code.qt.io/qt/qtwebkit.git",
+                    "branch" : "5.212"
+                }
+            ]
+        },
+        {
+            "name" : "qgis",
+            "buildsystem" : "cmake-ninja",
+            "sources" : [
+                {
+                    "type" : "git",
+                    "url" : "https://github.com/qgis/QGIS.git",
+                    "branch" : "final-3_2_1"
+                },
+                {
+                    "type" : "patch",
+                    "//source" : "https://issues.qgis.org/attachments/download/13041/qgis-3.2.1-fix-sip-name.patch",
+                    "path" : "qgis-3.2.1-fix-sip-name.patch"
+                }
+            ],
+            "config-opts" : [
+                "-DQWT_LIBRARY=/app/lib/libqwt.so"
+            ],
+            "modules" : [
+                {
+                    "name" : "proj",
+                    "sources" : [
+                        {
+                            "type" : "archive",
+                            "url" : "https://github.com/OSGeo/proj.4/releases/download/5.1.0/proj-5.1.0.tar.gz",
+                            "sha256" : "6b1379a53317d9b5b8c723c1dc7bf2e3a8eb22ceb46b8807a1ce48ef65685bb3"
+                        }
+                    ]
+                },
+                {
+                    "name" : "geos",
+                    "sources" : [
+                        {
+                            "type" : "archive",
+                            "url" : "http://download.osgeo.org/geos/geos-3.6.0.tar.bz2",
+                            "sha256" : "1fe7644f3240c8422b0143830ff142e44e8b01333c12f67681ccaab92142f2ea"
+                        }
+                    ]
+                },
+                {
+                    "name" : "spatialindex",
+                    "sources" : [
+                        {
+                            "type" : "archive",
+                            "url" : "http://download.osgeo.org/libspatialindex/spatialindex-src-1.8.5.tar.bz2",
+                            "sha256" : "31ec0a9305c3bd6b4ad60a5261cba5402366dd7d1969a8846099717778e9a50a"
+                        }
+                    ]
+                },
+                {
+                    "name" : "spatialite",
+                    "config-opts" : [
+                        "--enable-libxml2"
+                    ],
+                    "sources" : [
+                        {
+                            "type" : "archive",
+                            "url" : "http://www.gaia-gis.it/gaia-sins/libspatialite-sources/libspatialite-4.3.0a.tar.gz",
+                            "sha256" : "88900030a4762904a7880273f292e5e8ca6b15b7c6c3fb88ffa9e67ee8a5a499"
+                        }
+                    ],
+                    "modules" : [
+                        {
+                            "name" : "freexl",
```

Comment:
This doesn't build on aarch64. It needs it's configure regenerated https://flathub.org/builds/#/builders/39/builds/301 an example of doing this is https://github.com/flathub/shared-modules/blob/08e9b9eead7a95139be5d1fbc24bea730fb78f7a/SDL/SDL_net-1.2.8.json

Reply:
Isn't it what's being done few lines below?

Reply:
Not for spatialite, that I can see in the manifest https://flathub.org/builds/#/builders/37/builds/358/steps/5/logs/stdio

Reply:
ah, should be fixed for spatialite now